Applications/SimpleShot/layers/centorid_nearest_neighbors.cpp

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file   nearest_neighbors.cpp
+ * @date   09 Jan 2021
+ * @brief  This file contains the simple nearest neighbors layer, this layer
```

Comment:
How about move the description of this layer from @brief to @details?

Reply:
Done